### PR TITLE
Expand api docs

### PR DESCRIPTION
**Pull request by hatchan**

### CODE REVIEW

File: content/api/endpoints/builds.md [hunk 1]
```diff
@@ -1,46 +1,52 @@
 ## Builds
 
-Proin tincidunt, sem sit amet porttitor vulputate, `odio nisi gravidaz` nisi,
-[link](url). quis pharetra odio orci et mi.
+* [Get an build](#get-an-build)
```
Comment (mies) [inline content/api/endpoints/builds.md]: *a* build (not an)

File: content/api/endpoints/builds.md [hunk 2]
```diff
@@ -1,46 +1,52 @@
...
 
-* [chapter1](#chapter1)
-* [chapter2](#chapter2)
+### <a name="get-an-build"></a>Get an build
```
Comment (mies) [inline content/api/endpoints/builds.md]: *a* build (not an)

File: content/api/endpoints/deploys.md [hunk 3]
```diff
@@ -1,11 +1,60 @@
-# Deploys
-Whenever a deploy is triggered, wercker creates a dpeloy with a new id. Deploys
-start with a status of `not started` and then proceed to be `finished`. When a
-build is finished it has a result field that is either `passed` or `failed`.
+## Deploys
 
-| method | route                                    | description |
-| ------ | -----                                    | ----------- |
-| GET    | `app.wercker.com/api/v3/deploys/:deploy` | get a deploy with id `:deploy`
+* [Get an deploy](#get-an-deploy)
```
Comment (mies) [inline content/api/endpoints/deploys.md]: *a* deploy (not an)

File: content/api/endpoints/deploys.md [hunk 4]
```diff
@@ -1,11 +1,60 @@
...
 
-### GET deploys/:deploy
-Retrieve a deploy with the specified id.
+### <a name="get-an-deploy"></a>Get an deploy
```
Comment (mies) [inline content/api/endpoints/deploys.md]: *a* deploy (not an)

File: content/api/index.md [hunk 5]
```diff
@@ -1,3 +1,99 @@
 ## API Reference
 
-The wercker API is based on REST.
+Welcome to the documentation of the wercker API. Before starting to use the API,
+be sure to check out this document. Also checkout the getting started guides,
```
Comment (mies) [inline content/api/index.md]: again 'checkout' so: "Also have a look at the getting..."

File: content/api/index.md [hunk 6]
```diff
@@ -1,3 +1,99 @@
...
+such as [authentication](gettings-started/authentication.html), and
+[clients](gettings-started/clients.html)
+
+The current version of the API should be considered to be a _alpha_ version.
```
Comment (mies) [inline content/api/index.md]: an _alpha_

File: content/api/index.md [hunk 7]
```diff
@@ -1,3 +1,99 @@
...
+Though we will try to keep it backwards compatible, we cannot make that
+gaurentee at this moment.
```
Comment (mies) [inline content/api/index.md]: guarantee

File: content/api/index.md [hunk 8]
```diff
@@ -1,3 +1,99 @@
...
+path needs to be the following:
+
+```
+https://app.wercker.com/api/v3/
+```
+
+### Conventions
+
+The wercker API will use a couple of conventions. These next section will
+describe these convetions.
```
Comment (mies) [inline content/api/index.md]: conventions

File: content/api/index.md [hunk 9]
```diff
@@ -1,3 +1,99 @@
...
+All endpoints in the documentation use relative paths. The host and the prefix
+path needs to be the following:
+
+```
+https://app.wercker.com/api/v3/
+```
+
+### Conventions
+
+The wercker API will use a couple of conventions. These next section will
```
Comment (mies) [inline content/api/index.md]: "uses several conventions"

File: content/api/index.md [hunk 10]
```diff
@@ -1,3 +1,99 @@
...
+describe these convetions.
+
+#### Representations
+
+We currently have two ways of representing a resources: detailed and summary.
```
Comment (mies) [inline content/api/index.md]: a resource (no "s")

File: content/api/index.md [hunk 11]
```diff
@@ -1,3 +1,99 @@
...
+Where summary will contain a subset of information of the detailed
+representation.
+
+The summary representation will be used in lists or when embedded in a other
```
Comment (mies) [inline content/api/index.md]: another

File: content/api/index.md [hunk 12]
```diff
@@ -1,3 +1,99 @@
...
+  "message": "You are not authorized to access the resource",
+  "error": "Unauthorized",
+  "statusCode": 401
+}
+```
+
+The `error` and `statusCode` contain information related to the HTTP status
+code. The `message` contains a specific message related to the error.
+
+If a incorrect parameter or incorrect payload was used, we also include a
```
Comment (mies) [inline content/api/index.md]: an incorrect

**Comment by mies** [comment]
@hatchan looks good apart from some grammar. I did push some "an -> a" fixes yesterday (top of the PR) but they must have not been picked up?

**Comment by hatchan** [comment]
please don't add commits

**Comment by hatchan** [comment]
Thanks for the feedback. Processed the comments mentioned in this PR. 

I must have forced pushed over the other commits, since I thought I was working on this branch alone :-/

**Comment by mies** [comment]
@hatchan no worries, were some small typos. This looks very, so ready to rock!

**Pull request merged**